packages/models/user_profiles.coffee

```diff
@@ -0,0 +1,24 @@
+UserProfiles = new Mongo.Collection('userProfile')
+UserProfile = Astro.Class
+  name: 'UserProfile'
+  collection: UserProfiles
+  transform: true
+  fields: 
+    fullName: 'string'
+    jobTitle: 'string'
+    bio: 'string'
+    emailHidden: 'boolean'
+    userId: 'string'
+
+  methods:
+    update: (fields, callback) ->
+      this.set(fields)
+      this.save ->
+        callback?()
```

Comment:
Is this specific to Astro? With a meteor method you wouldn't need to - I think it would automatically call the callback with the return value of the method.